Documentation/Blazorise.Docs/Blazorise.Docs.csproj

```diff
@@ -56,13 +56,13 @@
     <ProjectDocsCompiler>dotnet run --configuration release --project "..\..\Documentation/Blazorise.Docs.Compiler/Blazorise.Docs.Compiler.csproj"</ProjectDocsCompiler>
   </PropertyGroup>
 
-  <!--Execute the code generator-->
-  <Target Name="CompileDocs" BeforeTargets="BeforeBuild">
-    <!--Command-line for the code generator-->
-    <Message Text="Generating Docs and Tests" Importance="high" />
-    <Exec Command="dotnet &quot;$(BinDocsCompiler)&quot;" Condition="Exists('$(BinDocsCompiler)')" />
-    <Exec Command="$(ProjectDocsCompiler)" Condition="!Exists('$(BinDocsCompiler)')" />
-  </Target>
+  <!--  &lt;!&ndash;Execute the code generator&ndash;&gt;-->
```

Comment:
Is this intentional commented to be removed, or do we need to keep it?

Reply:
I think I committed the `Blazorise.Docs.csproj` changes by mistake. I have commented out some pieces to speed up the build process. Any changes in `Blazorise.Docs.csproj` should be reverted.